src/otoole/cli.py

```diff
@@ -194,7 +196,7 @@ def conversion_matrix(args):
 
     if read_strategy and write_strategy:
         context = Context(read_strategy, write_strategy)
-        context.convert(args.from_path, args.to_path)
+        context.convert(args.from_path, args.to_path, input_data=input_data)
```

Comment:
Example of question 2, showing the need to pass in `input_data` through to the write strategy.